website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -1288,15 +1399,15 @@ The grafted subgraph can use a GraphQL schema that is not identical to the one o
 
 > **[Feature Management](#experimental-features):** `grafting` must be declared under `features` in the subgraph manifest.
 
-## File Data Sources
+## IPFS/Airweave File Data Sources
```

Comment:
Should be Arweave not Airweave. 
Link to project if you're curious: https://www.arweave.org/

Reply:
Oops! yes